src/frontend/src/flows/authorize/index.ts

```diff
@@ -188,6 +188,14 @@ const authenticate = async (
     };
   }
 
+  let autoSelectIdentity = undefined;
+  if (nonNullish(authContext.authRequest.autoSelectMatchingIdentity)) {
+    autoSelectIdentity = await getAnchorByPrincipal({
```

Comment:
Same here, this is an identity, now? Why call it `auto`?

Reply:
Well, yes. `getAnchorByPrincipal` resolves the supplied Principal to an identity (number).